app/views/layouts/_header.html.erb

```diff
@@ -1,8 +1,11 @@
 <%= render './layouts/headness.html.haml' %>
 <% if user_signed_in? && current_user.admin %>
   <% if api_status == 'failed' %>
-    <div class="grid w-full bg-yellow-550 text-sm place-content-center py-1">
-      The Classrooms Update Script failed to run today. Please report the issue.
+    <div class="flex flex-row justify-center items-center bg-red-800 text-red-100 text-sm place-content-center py-1">
```

Comment:
Do we need all three following classes  here ?

- justify-center
- items-center
- place-content-center

Reply:
I agree that if you place a div around the actual message block and add a centering you can do it with fewer classes.